documentation/add/07-sqw_redesign.md

```diff
@@ -141,29 +141,59 @@ The `Axis` classes describes image axes
+| *get_bin_nodes* | accepts the hypercube, produced by *get_axes_scales* and generates the grid which covers all range, described by the current **axes_block** class | |
+| *bin_pixels* | takes the coordinates of the pixels, expressed in the **access_block** coordinate system and, optionally, signal and error for these pixels and calculates  *npix*, *signal* and *error* from these pixels to the **axes_block** cells |  |
 
-The `ProjectionManager` class will manage all projection operations.
 
-![Projection Class Overview](../diagrams/projection.png)
+The particular projection classes are the children of the abstract **aProjection** class defining three abstract **aProjection** methods, which should be defined by child classes.
+These methods are:
 
-Responsible for all image projections - this includes symmetrization and simpler data operations:
+| method | Description | Notes |
+|-----|---------|---|
+| *transform_pix_to_img* | Transforms the points with the coordinates in the Crystal Cartesian coordinate system into the Image cordinate system | For **ortho_proj** class image coordinate system would be hkl, for **spher_proj**:  *Q,theta,phi,dE* |
+| *transform_img_to_pix* | Transform the points with the coordinate in the Image coordinate system into Crystal Cartesian coordinate system. | Operation, opposite to *transform_pix_to_img* | 
+| *get_axes_block* | returns the particular **axes_block** class, specific for the coordinate system, defined by the specific projection class | |
 
-- Crystal Cartesian to h,k,l,
-- arbitrary rotation,
-- arbitrary offset,
-- plane cuts
-- spherical cuts
-- cylindrical cuts
+Other important methods, necessary for the analysis of different cuts from an sqw object, and implemented using these transformation can be defined using these three abstract methods. These methods together with their short descriptions provided in the table:
+
+| method | Description | Notes |
+|-----|---------|---|
+| *from_cur_to_targ_coord* | Transforms the pixels of the current image coordinate system into other coordinate system the requested cut is presented in. | |
+| *get_nrange* | return ranges of indexes of pixels which may contribute into current cut | |
+| *bin_pixels* | transform input pixels into the coordinate system, defined by the target projection and calculate the contribution of these pixels into the areas, defined by target **axes_block** method. (namely *npix*, *s* and *err* arrays)| |
+
+The children of **aProjection** class (e.g. **ortho_proj** class, defining orthogonal projection), need to define the abstract methods and may redefine these methods for particular pairs of projections to optimize the performance of cuts.
+
+**Notes**
+^1 e.g., if **axes_block** defines a hypercubic grid with side **a**, the size of the scaler cube with respect to **orhto_projection** (see below) would be the **a/sqrt(2)** |
```

Comment:
typos: scaler->scalar, orhto->ortho; delete "the" before a/sqrt...

Does "see below" refer to the axes block overload requirement next or something else? It would be good to be more specific here.

Reply:
modified description but not entirely happy with the result. Leave this to the doc ticket.